pyfar/dsp/dsp.py

```diff
@@ -747,7 +747,10 @@ def regularized_spectrum_inversion(
     ----------
     signal : Signal
         The signals which spectra are to be inverted.
-    freq_range : tuple, array_like, double
+    frequency_range : tuple, array_like, double
+        ``'freq_range'`` parameter will be deprecated in pyfar 0.8.0 in favor
+        of ``'frequency_range'``.
```

Comment:
```suggestion
```
this need to go to the end

there you will add an other parameter ``freq_range``this will be same as ``frequency_range``but with the warning.